### DISCUSSION TITLE
Show & tell

### DISCUSSION THREAD

**Discussion by orangemug** [maputnik/editor#876] [Show and tell]
Hi Maputnik community,

I think it'd be great if we could share the work being created by all the talented people using Maputnik.

If you're a company, organisation or individual please share your work. What I'm looking for

 - Images / links to maps made with Maputnik
 - Testimonials - Why is Maputnik useful for you and your team?

It'd also be great to get a list of companies and organisations using Maputnik. Later we can publish this to the homepage.

So please share all your creations in the comments below.

**Comment by orangemug** [comment]
@pjsier @chriswhong @bartvde I thought you all might want to share your work. If you're not interested, feel free to ignore

**Comment by bartvde** [comment]
@orangemug we have actually taken a different approach since and are not using maputnik anymore

**Comment by orangemug** [comment]
@bartvde ok, thanks for the reply.

**Comment by pjsier** [comment]
@orangemug I work with an organization called Eviction Lab on an open source web app visualizing eviction data across the United States (more information here https://preview.evictionlab.org/). It's currently in development ([GitHub org here](https://github.com/EvictionLab)), but Maputnik has been great for making it easier for designers and developers to collaborate on map styles. It's been especially useful for us because we're using a lot of data driven styles and custom tilesets

Side note on that: when will the recent PRs be deployed to maputnik.github.io?

**Comment by orangemug** [comment]
Thanks @pjsier.

> Side note on that: when will the recent PRs be deployed to maputnik.github.io?

Yeah apologies I've been a bit slow on that I'll try to get on to that tomorrow. On yet another related note @lukasmartinelli can you make @pjsier a member of the maputnik org

**Comment by pjsier** [comment]
@orangemug no problem! Is there an issue out there for automating deployments with Travis? And, out of curiosity, is editor.openmaptiles.org pulling from the same source or is that updated separately?

**Comment by lukasmartinelli** [comment]
> Yeah apologies I've been a bit slow on that I'll try to get on to that tomorrow. On yet another related note @lukasmartinelli can you make @pjsier a member of the maputnik org

Done.

**Comment by chriswhong** [comment]
Hello all, we are a new dev shop at the New York City Department of City Planning.  We're building a lot of map-heavy applications, and maputnik has been a huge help.  You can see our first project that uses mapboxGL here: [https://communityprofiles.planning.nyc.gov](https://communityprofiles.planning.nyc.gov)

We also created [maputnik-dev-server](https://github.com/nycplanning/labs-maputnik-dev-server) as a way to quickly load whatever map is open in our dev environment in maputnik.  This workflow has proven very effective for us:
Load sources and layers with basic style into the map in our app > use maputnik-dev-server to load the current map style into maputnik > make style improvements > copy and paste JSON back into our codebase, repeat.

Our current apps involve some complex layer toggling UI and lots of different map states, so it's good to be able to send whatever the current map style is at any point in the development process over to maputnik quickly.

**Comment by orangemug** [comment]
From <https://github.com/maputnik/editor/issues/177#issuecomment-341619376>

> I just finished maptime seattle last night and used https://github.com version. it's little cumbersome but when you load the source data, I needed to disabled the security on the browser in Firefox to render the http source vector data.
I also used the same method to see preview in bl.ocks.org as well...all went fantastically smooth and students loved using Maputnik to see vector tiles being stylized and able to view the web map w/ any style they wanted!

Thanks @keum!

**Comment by keum** [comment]
Hello everyone and @orangemug 
My co-worker from King County GIS and I put together a single day workshop on generating vector tiles and styling with Maputnik last month at Washington State GIS conference in Olympia, WA. Here are two pictures from the workshop. Nothing very large but folks were very new to open source tools and almost everyone uses ESRI at their government agencies so it was great to share and expose various FOSS4G tools to them. Just wanted to share this with Maputnik community. Also we created a website w/ instruction [here](https://pkgeo-org.github.io/). We're going to finish updating the last section within next few weeks but everything else should be good. 
Would like to create sort of userguide workbook for using Maputnik down the road. 
thanks everyone for contributing and using this awesome tool.

Summary workflow diagram I made for the workshop
![screen shot 2018-06-09 at 10 03 01 am](https://user-images.githubusercontent.com/1133652/41194183-62071eb0-6bcc-11e8-9bc6-5ca85d40ce15.png)


![img-8226](https://user-images.githubusercontent.com/1133652/41194037-5d01d240-6bca-11e8-82d9-9ed98d6915d8.JPG)

**Comment by orangemug** [comment]
Very cool @keum, thanks for sharing!

**Comment by pathmapper** [comment]
This is not from me, but someone else presented a short talk at FOSSGIS 2018.

It's in German language, here is "Maputnik in Action":
https://media.ccc.de/v/2018-5370-webmapping_aber_mit_style#t=214

**Comment by oterral** [comment]
Basically here @swisstopo, we try to automatize all the process describe by @keum in this workshop.

Only 1 small difference,  we want to share styles created by maputnik with tileserver-gl via a github repository to have versioning and to allow reviews from leaders.

Here is our [styles repository](https://github.com/geoadmin/config-tileservergl). And our tileserver-gl will read soon from this repo.

We have also have bit like the [orangemug/maputnik-service](https://github.com/orangemug/maputnik-services),  including tileserver-gl and maputnik in a docker-compose and rancher configs [here](https://github.com/geoadmin/service-tileservergl).

Soon we plan to add the gatekeeper or micro-github in it. We are at the early stages.

To finish, maputnik needs to be available on the web and not as desktop because we plan to give access to our maputnik to external clients so they can style their own data. and we can't ask them to install something.

**Comment by keum** [comment]
@oterral that looks great, would love to hear when you are making it available.

**Comment by orangemug** [comment]
The documentation for [OS Open Zoomstack Vector Tile API](https://www.ordnancesurvey.co.uk/business-and-government/products/os-open-zoomstack.html) contains a section called "Using the Vector Tile API in Maputnik" 🎉 Which I think is pretty damn cool.

Docs <https://www.ordnancesurvey.co.uk/docs/user-guides/os-open-zoomstack-vector-tile-api.pdf>

I'd love to know what Ordnance Survey use Maputnik for internally?

CC @charleyglynn

**Comment by pathmapper** [comment]
From: https://developer.tomtom.com/maps-sdk-web/tutorials-advanced/creating-custom-vector-map-style

> There are several ways to edit styles. You may do this manually in a text editor of your choice, but we prefer to use Maputnik - a visual map designer.

**Comment by orangemug** [comment]
@jonahadkins, at a career day, with what looks like Maputnik https://twitter.com/jonahadkins/status/1121803336024502277 🎉

**Comment by jonahadkins** [comment]
> @jonahadkins, at a career day, with what looks like Maputnik https://twitter.com/jonahadkins/status/1121803336024502277 🎉

hi! I've been using a maputnik for a few years - professionally I use it to create styles for clients using mapbox & esri vector tiles - the above tweet was from my youngest daughters career day event - it's great as a learning tool because it's supports 3D/extrusions and I can easily show someone how to change colors - just those 2 things seem to be make anyone a believer in maps - I also recently did an event using Maputnik with the local code 4 america chapter, which [you can read about here](http://www.code4hr.org/2019/05/12/camp-digital-launch-party/).

**Comment by orangemug** [comment]
Thanks @jonahadkins great to hear where Maputnik is used. Please feel free to post anything else you create using Maputnik in this ticket.

Maybe your next Atlas of Design submission will be a map created using Maputnik 😄

Reference: For anyone not aware turn to page 30 of Atlas of Design volume 4 - https://www.jonahadkins.com/photoshop/openstreetmap/cartography/2018/05/19/atlas-of-design.html (that map's not made using Maputnik)

**Comment by egil1978** [comment]
The GeoDataViz team at Ordnance Survey (OS) have been using Mapnutik for a number of workshops we have run this year supporting the use of OS Open Zoomstack. Last night we were in Cambridge at Barclays Eagle Lab using Maputnik and github to create a Brewdog location map.
![20190611_194613](https://user-images.githubusercontent.com/6939088/59299836-8f0c5580-8c85-11e9-813a-e6d48af72d0b.jpg)

**Comment by orangemug** [comment]
That's awesome @egil1978, thanks for the photo!

**Comment by egil1978** [comment]
No problem!

**Comment by wilhelmberg** [comment]
Landing here by invitation from @orangemug in https://github.com/maputnik/editor/issues/554#issuecomment-541017484

A few years ago (2016) I and @prebm styled a map with Maputnik, solely made of OGD from the City of Vienna (https://www.data.gv.at/auftritte/?organisation=stadt-wien).

It's hosted at Mapbox now and the style says Mapbox all over, but it was completely developed locally with one of the early versions of Maputnik:

http://nicemap.at/


![image](https://user-images.githubusercontent.com/4549888/66648701-7ceaf580-ec2c-11e9-8ab1-cab1439bcce4.png)

![image](https://user-images.githubusercontent.com/4549888/66648704-807e7c80-ec2c-11e9-9059-194caf96c140.png)

![image](https://user-images.githubusercontent.com/4549888/66648714-83796d00-ec2c-11e9-90b4-ada58069c389.png)

**Comment by orangemug** [comment]
https://twitter.com/lukmartinelli/status/1198035921485979648 looks like TomTom use a fork of Maputnik now on their website now

**Comment by orangemug** [comment]
A nice blog post from @yasoob using Maputnik for map styling https://yasoob.me/posts/custom-map-with-tileserver-gl/

**Comment by kylebarron** [comment]
I've been using Maputnik to design a topographic fork of OSM Liberty, and it's been great! 

Repo: https://github.com/nst-guide/osm-liberty-topo 
Preview: https://nst-guide.github.io/osm-liberty-topo/#12/37.7456/-119.5936

**Comment by orangemug** [comment]
Nice work @kylebarron, that looks really good! Anything you think could be improved please raise an issue, suggested improvements are always welcome.

**Comment by kylebarron** [comment]
I have hit a few bugs, namely https://github.com/maputnik/editor/issues/565 and https://github.com/maputnik/editor/issues/588, and https://github.com/maputnik/editor/issues/223 would be a nice addition. I've generally been using the desktop executable with `--watch` so that I can edit the JSON in my text editor when I want to use an expression.

**Comment by orangemug** [comment]
Some more companies using Maputnik

 - [Elastic](https://www.elastic.co/) have a fork of Maputnik https://github.com/elastic/ems-basemap-editor
 - A tutorial from IGN https://geoservices.ign.fr/documentation/services_betas/Maputnik.html

**Comment by orangemug** [comment]
Also Mapcreator (Maps4News) also have a fork https://github.com/MapcreatorIO/editor (see diff here https://github.com/maputnik/editor/compare/master...MapcreatorIO:master), I have no idea what/how they use Maputnik, or wether they are just experimenting, maybe @RobbertK or @x0rz3q can comment?

**Comment by nspringer-trimble** [comment]
I am the Director of Cartography for [Trimble MAPS](https://maps.trimble.com) and we provide maps, navigation, geocoding, and other web services for the transportation industry.  We have a vector and raster map stack that generally uses the Mapbox formats (MBTiles, style spec, etc).

I use Maputnik with the desktop wrapper to makes changes to our many style files so that I can have realtime feedback with Visual Studio Code as I make changes.

**Comment by orangemug** [comment]
<img width="1020" alt="1000stars" src="https://user-images.githubusercontent.com/235915/82901485-e3bbe580-9f55-11ea-90aa-b8a03fc28aeb.png">

**1000** stars on GitHub 🎉

That's a milestone, congrats everybody who's worked on it over the years.

**Comment by orangemug** [comment]
<img width="368" alt="Screenshot 2020-07-17 at 09 23 09" src="https://user-images.githubusercontent.com/235915/87765195-70c14380-c80f-11ea-8e01-c06d8c887eb0.png">

Very cool 🥶

**Comment by pathmapper** [comment]
> Arctic Code Vault Contributor 

only because of [this](https://user-images.githubusercontent.com/235915/59569605-d62e8800-9083-11e9-9502-a15daa74d393.png) (from https://github.com/maputnik/editor/pull/529) :wink:

**Comment by orangemug** [comment]
<img width="636" alt="Screenshot 2020-09-09 at 14 22 36" src="https://user-images.githubusercontent.com/235915/92604117-0b864d80-f2a8-11ea-996c-16b8fb7f7bc8.png">

Source https://donate.openstreetmap.org/

Ping @pathmapper

**Comment by pathmapper** [comment]
:tada: :tada: :tada: 

Looks like the donation is related to this app: https://allride-app.com/  (CEO with the same name)

@orangemug did you run a script or how did you notice? 1st donation to list at https://maputnik.github.io/donate :rocket:

**Comment by orangemug** [comment]
> Looks like the donation is related to this app: https://allride-app.com/ (CEO with the same name)

I tried to find that, clearly you are far better at googling that me

> We’ll run a script periodically to pull down the latest donations that mention Maputnik.

@pathmapper that statement from the donate page may be a little bit of a lie, there is no script at the moment, I'll write one now though 😁 

I just do a `ctrl+f` on the openstreetmap page for 'Maputnik' occasionally.

**Comment by pathmapper** [comment]
> I just do a ctrl+f on the openstreetmap page for 'Maputnik' occasionally.

Glad you never gave up :-)

**Comment by orangemug** [comment]
Luxemburg's official geoportal has an option to open/style directly in Maputnik. Nice use of Maputnik

<img width="1280" alt="Screenshot 2020-09-19 at 17 02 15" src="https://user-images.githubusercontent.com/235915/93671338-ff657180-fa99-11ea-9a69-c1552ca3065e.png">

https://map.geoportail.lu/theme/main?version=3&zoom=9&X=696260&Y=6416103&lang=en&layers=747-540-1691&opacities=1-1-1&bgLayer=basemap_2015_global

**Comment by orangemug** [comment]
A video explaining how to edit styles with Maputnik https://www.youtube.com/watch?v=OGIg8_WwIYE

**Hint**: You can get youtube to auto translate subtitles, although because they are autogenerated 'JSON' -> 'Jason' and 'Maputnik' is 'Nick'... also yes we should have a Pidgeon mascot called Nick 😁

**Comment by orangemug** [comment]
@almccon from Stamen mentioning Maputnik on episode 71 of the Geomob Podcast (https://thegeomob.com/podcast/episode-71) just made my day. If you care to share any more deets @almccon please feel free to comment.

**Edit**: It's a one word mention at _22:50_ which yes... is enough make my day 😁

**Comment by orangemug** [comment]
Two talks at this years FOSS4G mention Maputnik 🎉

 - https://callforpapers.2021.foss4g.org/foss4g2021/talk/Q9VXD3/
 - https://callforpapers.2021.foss4g.org/foss4g2021/talk/NEMA9F/

**Comment by pathmapper** [comment]
Three talks at FOSSGIS 2021 mentioned Maputnik (talks in German language, but Maputnik is also mentioned on the slides)

- https://media.ccc.de/v/fossgis2021-9014-vector-tiles-stand-der-technik-2021#t=937
- https://media.ccc.de/v/fossgis2021-9044-vectortiles-mithilfe-von-tilemaker-erstellen-und-nutzen#t=1244

My personal favorite is the following slide from 
- https://media.ccc.de/v/fossgis2021-9010-smartmapping-amtliche-daten-als-vectortiles#t=827

![grafik](https://user-images.githubusercontent.com/20856381/128128597-fb6ac87f-8551-4ad5-be5b-462f95a66245.png)

**Comment by propheel** [comment]
Hi all, I was asked by @pathmapper to share my story with Maputnik :grin:

I work for @JaneaSystems and want to extend [Azure Maps](https://azure.microsoft.com/en-us/services/azure-maps) product with custom styling functionality for the customers. There was an initial effort done in [Oct 2021](https://github.com/ambientlight/editor) which I got inspired by and want to extend Maputnik editor to also use [Azure Maps API](https://docs.microsoft.com/en-us/rest/api/maps) especially [Creator](https://docs.microsoft.com/en-us/rest/api/maps-creator).

There is a bunch of [Azure Maps examples](https://github.com/Azure-Samples/AzureMapsCodeSamples) if you want to see the product.

In the meantime I read the story behind this project and seeing it in action I decided to participate a little bit more than necessary. Hope it will benefit all :wink:

Kudos to all contributors for awesome work! 🚀

**Comment by CommanderStorm** [comment]
[`maplibre/martin`](https://github.com/maplibre/martin) will ship with martin build in in the next release.
Initial integration is a bit rough, but is is a start:

<img width="1570" height="937" alt="image" src="https://github.com/user-attachments/assets/aa32b12c-e4b5-42ce-b924-aa6d24be7ea1" />
<img width="1883" height="901" alt="image" src="https://github.com/user-attachments/assets/ee990995-4c6d-4dcb-932f-43a43ede86c0" />
